jazz_deployments/index.js

```diff
@@ -30,7 +30,7 @@ const crud = require("./components/crud")(); //Import the crud module.
 const request = require('request');
 const util = require('util');
 
-module.exports.handler = (event, context, cb) => {
+function handler (event, context, cb) {
```

Comment:
Can we just use the standard convention we have in the rest of our code?
`var x = function() { ...`

Reply:
This does not work when you have to stub it.